example/CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ add_executable(get_sound_speed_press
 target_link_libraries(get_sound_speed_press PRIVATE
   singularity-eos::singularity-eos)
 
+add_executable(Benchmark
```

Comment:
probably don't want this capitalized as it will make the executable capitalized, which is not standard.